src/planscape/config/conditions.json

```diff
@@ -108,17 +108,17 @@
                   "raw_data_download_path": "sierra-nevada/airQuality/Tier2/PotentialSmokeModerateSeverity_2021.tif",
                   "normalized_layer": "sierra-nevada:airQualityTranslate_Tier2_PotentialSmokeModerateSeverity_2021_scored",
                   "normalized_data_download_path": "sierra-nevada/airQualityTranslate/Tier2/PotentialSmokeModerateSeverity_2021_scored.tif",
-                  "display_name": "Potential Smoke Emissions (moderate severity fire)",
+                  "display_name": "Potential Smoke Emissions - Moderate Severity Fire",
                   "source": "Sierra Nevada Regional Resource Kit",
                   "source_link": "https://data.fs.usda.gov/geodata/rastergateway/regional-resource-kit/index.php",
```

Comment:
To update the source link in the info card, this source_link value will have to be updated: 
https://github.com/OurPlanscape/Planscape/blob/170b499ba1daac5dd6fe58691dd67b7141578a02/src/interface/src/app/map/layer-info-card/layer-info-card.component.html#L14

(not actually sure what reference_link is for, since it seems like there's a  hasReferenceLink() check in layer-info-card.component.ts but the reference_link is never actually used in the html)

Reply:
Done!